stcgal/frontend.py

```diff
@@ -40,10 +41,12 @@ def __init__(self, opts):
             self.protocol = Stc12Protocol(opts.port, opts.handshake, opts.baud)
         elif opts.protocol == "stc15a":
             self.protocol = Stc15AProtocol(opts.port, opts.handshake, opts.baud,
-                                          round(opts.trim * 1000))
-        else:
+                                           round(opts.trim * 1000))
+        elif opts.protocol == 'stc15':
```

Comment:
Everywhere in stcgal double quotes are used. Please do that too, for consistency's sake.